index.html

```diff
@@ -1,11 +1,241 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>

```

Comment:
you forgot to add a meta tag:
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`
to make your website adaptive on small screens